[terra-clinical-item-view] Add property to separate by row or column when in two column layout (#912)
cerner/terra-clinical: packages/terra-clinical-item-view/CHANGELOG.md

## Unreleased

* Added

>   * Added `trueColumn` prop so consumers can choose whether to separate displays in the two column layout by row or by column.

Reviewer: ```suggestion
  * Added `trueColumn` prop to toggle between displaying in a two column layout by row or by column.
```

Author: Fixed: https://github.com/cerner/terra-clinical/pull/912/commits/abe1feec60bf2b34abcc990db0c2964416740638#diff-019a0d835127f70f44b926383f7f9ca20fa8848183637b56a0fe47a22104b394R6
